main/index.jsx

```diff
@@ -1,4 +1,18 @@
-// TODO: Create App Layout
-const Main = ({ children }) => children
+import React, { useRef } from 'react'
+
+import Head from './head'
+
+import { initStore, GlobalProvider } from 'store'
+
+const Main = ({ children }) => {
+  const store = useRef(initStore)
```

Comment:
This you pass a reference to the function. Is this what you want?

Reply:
```suggestion
  const store = useRef(initStore())
```